ScoutSuite/output/data/inc-scoutsuite/pagination.js

```diff
@@ -0,0 +1,162 @@
+var defaultPageSize = 2
+var regionPageSize = 1
+var reCount = new RegExp('_count+$')
```

Comment:
Why is there a `+`? This will match `_countttttttttt`. It probably won't cause any problem, but it seems like this is not necessary.

```suggestion
var reCount = new RegExp('_count$')
```

Reply:
You're right, @zer0x64 told me to, I'll remove it (I'm a noob at regex)